src/toast/_toast.scss

```diff
@@ -0,0 +1,58 @@
+.toast {
```

Comment:
We can remove these styles altogether I think

Reply:
I moved it under the `stories` directory to display colorful. Can it stay like thatthis? 
Also, CSS variable declaration can help create Toast component in the project.